CHANGELOG

```diff
@@ -18,5 +17,11 @@ Updated deprecated code in instance method "comments_ordered_by_submitted"
 	- changed "self.type.name" to "self.class.name"
 	
 v0.1.2
+------
+fixed a problem that arises when trying to use the acts_as_voteable plugin with your comments
 
-fixed a problem that arises when trying to use the acts_as_voteable plugin with your comments
\ No newline at end of file
+v1.3.0
+------
+- July 13, 2022
+- Update the app to work with Rails 6 and 7
+- Add support to run specs against multiple Rails versions
```

Comment:
I shifted the version number over so it reflects how the gemspec was initially referencing this. 

On the gemspec, 0.1.2 in the changelog was referenced as 1.2.0 (in gemspec). I suspect that the changelog formatting was not "in sync" in the original gem.